camera-config/MastCameraCalibration.yml

```diff
@@ -3,7 +3,7 @@
 name: "mast"
 description: "Camera located on the mast of the rover."
 # Make sure that this camera_id matches the one defined in the udev rules.
-camera_id: 20
+camera_id: 0
```

Comment:
Clarifying this, we set the camera ID to 20 since we created udev rules that would ensure this specific camera (identified by its USB vendor/device IDs) was always mounted at `/dev/video20` as well as whatever camera ID it was automatically assigned by the kernel. This means even with multiple cameras that might have different auto-assigned IDs each time, we could guarantee they all had stable IDs that wouldn't conflict with any others.
If the udev rules aren't modifed, we should probably revert this change. Thoughts?

Reply:
Udev rules are at [rover-config/50-rover-cameras.rules](https://github.com/huskyroboticsteam/Resurgence/blob/master/rover-config/50-rover-cameras.rules), and honestly should probably be updated to reflect the hardware we are using this year.

Reply:
Oh I wasn't aware of the udev rules, that's a good solution. The camera wasn't getting picked up on 20 so I changed the value to 0, but if we can update the udev rules then I agree this change should be reverted.

For now, until we can change those rules and test it, let's leave as is and add a to-do?